include/types/HttpRequest.hpp

```diff
@@ -0,0 +1,53 @@
+#ifndef HTTPREQUEST_HPP
+#define HTTPREQUEST_HPP
+
+#include <string>
+#include <any>
+#include <vector>
+#include <unordered_map>
+
+namespace api
+{
+    /**
+     * @struct HTTPRequest
+     * @brief This structure defines an example type for HTTP(S) reply.
+     */
+    struct HTTPRequest
+    {
+        /**
+         * @brief This string represents the URI that has been queried by user.
+         */
+        std::string uri;
+
+        /**
+         * @brief This string refers to the method used by the client to query the server.
+         * example: POST / GET / PUT ...
+         */
+        std::string method;
+
+        /**
+         * @brief This string refers to the request body.
```

Comment:
@brief Refers to [...]

Reply:
same as the previous one I'd go with only "The request body"